### ISSUE TITLE
Inspiration: Ben Shneiderman’s Creative Support Tools

### ISSUE THREAD

**Issue by skilfullycurled** [good-labs/slopi-communication#11]
First, you (and others) might be interested in (or already know about) Ben Shneiderman’s work on creative support tools.  He wrote one of my favorite books called Leonardo’s Laptop, and in it he proposes two ways of working that might be interesting to think about in a SLOPI context.  Shneiderman is trying to apply the creative process to all aspects of life under the idea that we create our lives so therefore living requires creative support tools.  SLOPI might want to be thought of in these larger life terms.  What are the activities that people do to create their lives and how might SLOPI communication be used in all of those aspects.  I've attached the sort of "thesis" images here but here's a link to the [full presentation](https://www.cs.umd.edu/hcil/pubs/presentations/LeonardoLaptop/LeonardoLaptop.ppt ).

<img width="716" alt="Screen Shot 2019-05-14 at 9 28 48 PM" src="https://user-images.githubusercontent.com/950291/57795551-414b0e80-7714-11e9-8fb0-5a3d09b400da.png">
<img width="716" alt="Screen Shot 2019-05-14 at 9 29 08 PM" src="https://user-images.githubusercontent.com/950291/57795556-4445ff00-7714-11e9-87db-f1ac33994ac3.png">

**Comment by pdurbin** [comment]
@skilfullycurled thanks for opening this issue!

> two ways of working

I flipped through the slides but I couldn't tell at a glance what the two ways of working are.

Does https://www.youtube.com/watch?v=S5jlKfXWA-w do the book justice? :smile: It's only six and a half minutes long. :smile:

**Comment by pdurbin** [comment]
@skilfullycurled I just watched that 6.5 minute video I linked above, "Leonardo's Laptop: Human Needs and the New Computing Technologies (2002)" at https://www.youtube.com/watch?v=S5jlKfXWA-w and I think I have a better idea of where Shneiderman is coming from. Also, those PowerPoint slides you linked at https://www.cs.umd.edu/hcil/pubs/presentations/LeonardoLaptop/LeonardoLaptop.ppt make a bit more sense to me. Here are some screenshots of the slides that seem to relate to SLOPI the most:

![Screenshot from 2019-05-26 08-59-52](https://user-images.githubusercontent.com/21006/58382152-ae706680-7f94-11e9-88e6-627e7475f7d2.png)

![Screenshot from 2019-05-26 08-59-55](https://user-images.githubusercontent.com/21006/58382151-ae706680-7f94-11e9-9748-36deefd8b541.png)

![Screenshot from 2019-05-26 08-59-57](https://user-images.githubusercontent.com/21006/58382150-ae706680-7f94-11e9-8061-f6a18320793d.png)

![Screenshot from 2019-05-26 09-00-00](https://user-images.githubusercontent.com/21006/58382149-ae706680-7f94-11e9-9b61-de4644f8b91c.png)

When I see I items like these...

- negotiated expectations
- million person community
- business strategy
- policy manifesto
- tell stories
- record events
- train novices
- report decisions
- mentorship
- preserve history
- leadership

... I think that there are opportunities for some of these items to be communicated within the context of a open source project in a SLOPI way.

Thanks for putting Leonardo's Laptop on my radar.

Do you think I'm starting to understand what you're getting at?

**Comment by skilfullycurled** [comment]
Sure.  It was just a thought.  I think the main thing was just that I thought it was interesting to take a number of aspects of a theory and to see how each one could apply to the "essential" aspects of life.

**Comment by skilfullycurled** [comment]
Also, I noticed that you assigned this issue to me.  I appreciate the "promotion" but I really was just interested in contributing to the discussion you started.  You had asked that I post my thoughts as issues so I did, but I'd prefer not to have any issues assigned to me.  Thanks though!

Hmmmm...it seems I can no longer comment.  I only have the option to close and comment.  In any case, I'll do that, but even though the comment is closed, I'd still like to be unassigned.  Thank you again for thinking of me!
